types/electron-settings/v2/electron-settings-tests.ts

```diff
@@ -1,4 +1,5 @@
-import * as settings from 'electron-settings';
+import settings = require('electron-settings');
+import Observer = require('electron-settings/lib/observer');
```

Comment:
I don't understand. Why expose this? It seems to be an implementation module and not part of the API.